wdtk-datamodel/src/test/java/org/wikidata/wdtk/datamodel/helpers/DatamodelConverterTest.java

```diff
@@ -225,7 +215,19 @@ public void testGenerationFromOtherItemDocument() {
 		DatamodelConverter converter = new DatamodelConverter(new DataObjectFactoryImpl());
 		assertEquals(item, converter.copy(item));
 	}
-	
+
+	@Test
+	public void testGetJsonId() throws Exception {
+		ItemDocument item = Datamodel.makeItemDocument(
+				Datamodel.makeWikidataItemIdValue("Q42"),
+				Collections.singletonList(Datamodel.makeMonolingualTextValue("en", "label")),
+				Collections.singletonList(Datamodel.makeMonolingualTextValue("en", "desc")),
+				Collections.singletonList(Datamodel.makeMonolingualTextValue("en", "alias")),
+				Collections.emptyList(),
+				Collections.singletonMap("enwiki", Datamodel.makeSiteLink("foo", "enwiki", Collections.emptyList())));
+		DatamodelConverter converter = new DatamodelConverter(new DataObjectFactoryImpl());
+		Assertions.assertEquals("Q42", ((ItemDocumentImpl) (item)).getJsonId());
```

Comment:
`assertEquals` is already statically imported, so we should be able to do:
```suggestion
		assertEquals("Q42", ((ItemDocumentImpl) (item)).getJsonId());
```